### PR TITLE
RIF relay for claim transactions

### LINKED ISSUE #614: RSK: RIF Relay Integration
This might be a massive feat, as the RIF Relay API is not meant to be used directly but with a provided client.: https://github.com/rsksmart/rif-relay-client/blob/main/src/RelayClient.ts

Tests: https://github.com/rsksmart/rif-relay/blob/main/test/relayserver/RelayServerNoClient.test.ts)
How to interact with the server without using the client (apart from the signature): https://github.com/rsksmart/rif-relay/blob/main/test/utils/NoClient.ts#L84
Comment: the flow may look complex, and that's why we implemented the client, but if you check the tests, you will see that only some fields would be actually required, while others can be filled using the code I’ve extracted (using https://github.com/rsksmart/rif-relay/blob/main/test/utils/NoClient.ts).

### PR DESCRIPTION
**Pull request by michael1011**
Closes #614, #327

Depends on: https://github.com/BoltzExchange/regtest/pull/29

~Only works on regtest for now.~

Todos:

- [x] populate testnet config
- [x] fix block explorer links
- [x] strip ABIs to only what is needed
- [x] @kilrau needs translations

### CODE REVIEW

**Comment by kilrau** [comment]
image
liquid still not enabled

**Comment by michael1011** [comment]
> image liquid still not enabled

Completely unrelated to the PR

**Review by kilrau — APPROVED** [review]
UA Tested

File: src/components/CreateButton.tsx [hunk 1]
```diff
@@ -164,6 +162,28 @@ export const CreateButton = () => {
...
+        if (assetReceive() === RBTC) {
+            const [balance, gasPrice] = await Promise.all([
+                signer().provider.getBalance(await signer().getAddress()),
+                signer()
+                    .provider.getFeeData()
+                    .then((data) => data.gasPrice),
+            ]);
+            log.debug("RSK balance", balance);
+
+            const balanceNeeded = gasPrice * GasNeededToClaim;
```
Comment (maybeast) [inline src/components/CreateButton.tsx:176]: when deploying for the first time should we also account for `GasNeededToDeploy` here?

**Review by maybeast — COMMENTED** [review]
concept ACK

**Review by kilrau — CHANGES_REQUESTED** [review]
EDIT: fixed on latest commit

**Comment by kilrau** [comment]
Starting to look good!!

**Review by maybeast — APPROVED** [review]

**Pull request merged**